napari_plugin_manager/qt_package_installer.py

```diff
@@ -230,7 +230,7 @@ def constraints() -> Sequence[str]:
         pin_level = 2 if is_dev else 3
         version = ".".join([str(x) for x in _napari_version_tuple[:pin_level]])
 
-        return [f"napari={version}"]
+        return [f"napari={version}", "pydantic<=2.0"]
```

Comment:
```suggestion
        return [f"napari={version}", "pydantic<2.0"]
```

Reply:
Then https://github.com/napari/napari/pull/5785 needs to be amended too.